Launcher/Launcher/src/Commands/Update.cs

```diff
@@ -0,0 +1,60 @@
+using System.CommandLine;
+using Launcher.Helpers;
+using WSL;
+
+namespace Launcher.Commands;
+
+public static class Update {
+    public static Command GetCommand() {
+        var command = new Command("update") {
+            Description = $"Update {DistributionInfo.DisplayName} to the version bundled with this launcher"
+        };
+
+        command.SetHandler(() => {
+            ulong Run() {
+                if (!StartupHelper.BootDistro()) return 1;
+
+                var tarball = InstallationHelper.FindTarball();
+                if (tarball == null) return 1;
+
+                var path = Path.GetDirectoryName(tarball);
+                if (path == null) return 1;
+                Directory.SetCurrentDirectory(path);
+
+                var tmpDir = "/tmp/nixos-wsl-updater/installer";
```

Comment:
We should rm that, just in case it exists or use mktmp

Reply:
I'm not really happy with the update process in general, but I haven't found a nicer way to handle it than unpacking the installer somewhere and subsequently unpacking the rootfs

mktemp sounds like a good idea here

Reply:
I have now chosen a combined approach: the directory name gets generated from the version number and `rm`ed before the unpack